README.md

```diff
@@ -89,6 +89,9 @@ The Azure MCP Server provides tools for interacting with the following Azure ser
 - JSON output formatting
 - Cross-platform compatibility
 
+### 🔧 Azure Native ISV Services
```

Comment:
Please also add your team to .github/CODEOWNERS following the pattern set there by KeyVault, Kusto, etc.  "%area-ISV"